src/rc.c

```diff
@@ -30,29 +33,29 @@ static void convertPWMtoRad()
   // Get Roll control command out of RC
   if (_rc_control.x.type == ANGLE)
   {
-    _rc_control.x.value = ((pwmRead(_params.values[PARAM_RC_X_CHANNEL]) - _params.values[PARAM_RC_X_CENTER])
-                           *2*_params.values[PARAM_RC_MAX_ROLL_MRAD])/_params.values[PARAM_RC_X_RANGE];
+    _rc_control.x.value = (float)((pwmRead(_params.values[PARAM_RC_X_CHANNEL]) - _params.values[PARAM_RC_X_CENTER])
+                           *2.0f*get_param_float(PARAM_RC_MAX_ROLL_MRAD))/(float)_params.values[PARAM_RC_X_RANGE];
```

Comment:
Probably just want to rename these params to get rid of the units now that they're in SI units (e.g. `PARAM_RC_MAX_ROLL` instead of `PARAM_RC_MAX_ROLL_MRAD`)